src/main/webapp/app/delete-dialog/delete-dialog.component.ts

```diff
@@ -0,0 +1,29 @@
+import { Component } from '@angular/core';
+import { NgbActiveModal } from '@ng-bootstrap/ng-bootstrap';
+
+@Component({
+    selector: 'jhi-delete-dialog',
+    templateUrl: './delete-dialog.component.html',
+})
+export class DeleteDialogComponent {
+    confirmExerciseName: string;
+    entityTitle: string;
+    deleteQuestion: string;
+    deleteConfirmationText: string;
+
+    constructor(public activeModal: NgbActiveModal) {}
+
+    /**
+     * Closes the dialog
+     */
+    clear() {
+        this.activeModal.dismiss('cancel');
+    }
+
+    /**
+     * Deletes specified entity and closes the dialog
```

Comment:
The doc is incorrect: It only closes the dialog with a 'confirm' message so the user of the service can use this message to delete the entity.

Reply:
True, good suggestion